perfkitbenchmarker/packages/fio.py

```diff
@@ -22,9 +21,12 @@
 GIT_REPO = 'git://git.kernel.dk/fio.git'
 GIT_TAG = 'fio-2.1.14'
 FIO_PATH = FIO_DIR + '/fio'
+FIO_CMD_PREFIX = '%s --output-format=json ' % FIO_PATH
 SECTION_REGEX = r'\[(\w+)\]\n([\w\d\n=*$/]+)'
 PARAMETER_REGEX = r'(\w+)=([/\w\d$*]+)\n'
 GLOBAL = 'global'
+CMD_SECTION_REGEX = r'--name=(\w+)\s+'
+JOB_SECTION_REGEX = r'[\1]\n'
```

Comment:
Please rename `JOB_SECTION_REGEX` to indicate that it's a replacement pattern, not a search pattern.